openmetadata-ui/src/main/resources/ui/src/components/DashboardDetails/DashboardDetails.component.tsx

```diff
@@ -581,7 +581,7 @@ const DashboardDetails = ({
                         {charts.map((chart, index) => (
                           <tr
                             className={classNames(
-                              'tableBody-row',
+                              'tableBody-row tw-border-l-0 tw-border-r-0 tw-border-b-0',
```

Comment:
Can't we directly modify the `tableBody-row` class?

Reply:
Made requested changes